sdk/tables/azure-data-tables/tests/test_table_entity_async.py

```diff
@@ -285,23 +285,12 @@ def _assert_merged_entity(self, entity):
         # self.assertIsNotNone(entity.Timestamp)
         # self.assertIsInstance(entity.Timestamp, datetime)
 
-    def _assert_valid_full_metadata(self, metadata):
+    def _assert_valid_metadata(self, metadata):
         keys = metadata.keys()
-        self.assertIn("client_request_id", keys)
-        self.assertIn("request_id", keys)
-        self.assertIn("version", keys)
-        self.assertIn("date", keys)
-        self.assertIn("etag", keys)
-        self.assertIn("preference_applied", keys)
-        self.assertIn("content_type", keys)
-
-    def _assert_valid_update_metadata(self, metadata):
-        keys = metadata.keys()
-        self.assertIn("client_request_id", keys)
-        self.assertIn("request_id", keys)
         self.assertIn("version", keys)
         self.assertIn("date", keys)
         self.assertIn("etag", keys)
+        self.assertEqual(len(keys), 3)
```

Comment:
do you want to add a test that there's no other metadata besides version, date, and etag? Also wondering how these three were the ones decided on

Reply:
There is other metadata but it is intentionally stripped away by @annatisch 's recommendation.

Reply:
i see, but can you test that the stripped away metadata isn't present in the final response object, or am I misunderstanding what actually gets stripped away